src/Cms/LanguageRoutes.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Kirby\Toolkit\F;
+
+class LanguageRoutes
+{
+    /**
+     * Inject all language routes into
+     * the given routes array
+     *
+     * @param App $kirby
```

Comment:
This needs an absolute class name (same below).

Reply:
✅